Praneeth/add seconary domains

arepraneeth opened the pull request.

praneeth-sq committed:
Adding support for secondary domains and ipsets

briansharon commented on README.md:
````diff
@@ -419,6 +446,88 @@ To delete a domain:
+```
+
+
+To retrieve secondary_domain_by_id:
````
nit: don't need underscores in the description

briansharon commented on lib/dnsmadeeasy.rb:
```diff
@@ -23,6 +23,9 @@ class InvalidCredentialsFormatError < Error;
   class NoSuchAccountError < Error;
   end
 
+  class NoDomain < StandardError;
```
nit: `NoDomainError` instead of just `NoDomain`

briansharon reviewed (changes requested):
Looks good to me. Just two minor comments for readability and naming.

praneeth-sq committed:
Changing sandbox url

Changing sandbox url to api.sandbox.dnsmadeeasy.com as per
https://api-docs.dnsmadeeasy.com/?version=latest#530f2ac5-f8c4-4e88-b510-a2f26818ab7a,
